packages/core/src/app/address/StaticAddress.scss

```diff
@@ -1,5 +1,18 @@
 .checkout-address--static {
+    $icon-size: 1.85rem;
```

Comment:
Sorry I missed this before.
Could you please move this setting to `packages/core/src/scss/settings/foundation/forms/_settings.scss` and provide more information in the varible's name?
Plus, I didn't see we use `1.85rem` before, can you please justify why this value is appropriate?

Reply:
This size is specific only to one icon PP Connect, and size 1.85 is regarding to design.
In case when it is a size only for one icon in specific case I don't think that we need to move it global settings.

Reply:
I'm aware that it's a setting just for address.
However, except in the setting file, we strongly oppose putting numbers into CSS code. By doing so, we can control all of these different sizes from a single location.
Perhaps we could name it `$paypal-connect-icon-size` and put it there.

Reply:
done